src/components/exchange/ExchangeAssetList.js

```diff
@@ -130,13 +129,10 @@ const ExchangeAssetList = ({ itemProps, items, onLayout, query }) => {
   );
 
   const renderItemCallback = useCallback(
-    ({ item, section: { title: sectionTitle } }) => (
+    ({ item }) => (
       <ExchangeCoinRow
         {...itemProps}
-        isVerified={
```

Comment:
Instead of relying on the section title to determine whether or not an item is verified, I just use the isVerified tag that is now included (e.g, when a favorite item is unverified, we still want to treat it as unverified even if it is not in the unverified section)